src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.c

```diff
@@ -1444,7 +1444,7 @@ void EnBigpo_DrawCircleFlames(Actor* thisx, PlayState* play) {
 
         gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(play->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
 
-        gSPDisplayList(POLY_XLU_DISP++, &gGameplayKeepDrawFlameDL);
+        gSPDisplayList(POLY_XLU_DISP++, gEffFire1DL);
```

Comment:
Ah, you can ignore the comments from the other PR that also removes the `&` from this same variable that's named (or you can remove it and save yourself some conflicts